themes/bootstrap3/js/common.js

```diff
@@ -271,18 +271,20 @@ var VuFind = (function VuFind() {
   };
 
   function setupQRCodeLinks(_container) {
-    var container = _container || $('body');
-
-    container.find('a.qrcodeLink').on('click', function qrcodeToggle() {
-      var holder = $(this).next('.qrcode');
-      if (holder.find('img').length === 0) {
-        // We need to insert the QRCode image
-        var template = holder.find('.qrCodeImgTag').html();
-        holder.html(template);
-      }
+    var container = _container || document.body;
+    var qrcodeLinks = container.querySelectorAll('a.qrcodeLink');
+    qrcodeLinks.forEach((link) => {
+      link.addEventListener('click', () => {
+        var holder = this.nextElementSibling;
```

Comment:
I tested this and it is not working for me. I suspect that this line may be the issue -- I'm not sure that `this.nextElementSibling` is equivalent to `$(this).next('.qrcode')` since it is not accounting for the requested class name. However, I could be wrong!

In any case, if you would like to test this yourself so you can troubleshoot more easily, just edit your `$VUFIND_LOCAL_DIR/config/vufind/config.ini` file and turn on `showInResults` in the `[QRCode]` section. Then when you perform a search, you'll see QR Code drop-downs on the right side of every search result. If you expand the drop-downs, you'll find that you see a QR code in dev, but in this branch the image does not load.

Reply:
Thanks @demiankatz, I will try it out.